entrypoint.sh

```diff
@@ -47,13 +59,34 @@ function split_certs() {
     fi
   done
 
-  mkdir -p /etc/haproxy/certs > /dev/null 2>&1
   rm /etc/haproxy/certs/cert*_gen.pem > /dev/null 2>&1 || true
   mv cert*_gen.pem /etc/haproxy/certs/
-  etcd_set "certs/bundle" "value=true"
   rm cert*_gen.pem > /dev/null 2>&1 || true
 }
 
+HAVE_CERTS=
+
+function load_certs() {
+  mkdir -p /tmp/ssl_certs
+  mkdir -p /etc/haproxy/certs
+
+  if [ -n "${SSL_CERTS}" ]; then
+    echo "[kontena-lb] splitting bundled certificates from SSL_CERTS..."
+    split_certs
+    HAVE_CERTS=true
+  fi
+
+  for certenv in "${!SSL_CERT_@}"; do
+    echo "[kontena-lb] loading bundled certificate from $certenv..."
+
+    cat > /tmp/ssl_certs/$certenv.pem <<<"${!certenv}"
+    load_cert "$certenv" /tmp/ssl_certs/$certenv.pem
```

Comment:
This assumes that each `SSL_CERT_*` env only contains a single cert bundle. They do not get split up, and I'm not sure how haproxy would behave if you did supplying multiple cert bundles in a single `SSL_CERT_*` env var.